From 9a4d027e446752cfb744fd9323aa6c47fa357c7b Mon Sep 17 00:00:00 2001
From: mohamed bilel khadhraoui <bkhadhraoui870@gmail.com>
Date: Mon, 13 Apr 2026 17:16:58 +0100
Subject: [PATCH] modified:   include/configs/stm32mp13_st_common.h

---
 include/configs/stm32mp13_st_common.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/include/configs/stm32mp13_st_common.h b/include/configs/stm32mp13_st_common.h
index a4d83050..a8ff4a8b 100644
--- a/include/configs/stm32mp13_st_common.h
+++ b/include/configs/stm32mp13_st_common.h
@@ -8,6 +8,9 @@
 #ifndef __CONFIG_STM32MP13_ST_COMMON_H__
 #define __CONFIG_STM32MP13_ST_COMMON_H__
 
+#define TILIKUM_BOOTCMD \
+	"tilikum_bootcmd=sysboot ${boot_device} ${boot_instance}:8 any ${pxefile_addr_r} /extlinux/extlinux.conf;"
+
 #define STM32MP_BOARD_EXTRA_ENV \
 	"usb_pgood_delay=2000\0" \
 	"console=ttySTM0\0"
@@ -22,6 +25,7 @@
 
 #define CFG_EXTRA_ENV_SETTINGS \
 	STM32MP_MEM_LAYOUT \
+	TILIKUM_BOOTCMD \
 	STM32MP_BOARD_EXTRA_ENV
 
 #endif
-- 
2.43.0

